openapi/whatsapp.business/client.bal

```diff
@@ -0,0 +1,500 @@
+        return response;
+    }
+    # Download Webhook CA Certificate
+    #
+    # + return - Certificate or else an error
+    remote isolated function downloadWebhookCaCertificate() returns string|error {
+        string  path = string `/certificates/webhooks/ca`;
+        string response = check self.clientEp-> get(path, targetType = string);
+        return response;
+    }
+    # Upload Webhook CA Certificate
+    #
+    # + payload - Certificate
+    # + return - HTTP response or else an error
+    remote isolated function uploadWebhookCaCertificate(string payload) returns http:Response|error {
+        string  path = string `/certificates/webhooks/ca`;
+        http:Request request = new;
+        request.setPayload(payload);
+        http:Response response = check self.clientEp->post(path, request, targetType=http:Response);
+        return response;
+    }
+    # Check-Contact
+    #
+    # + payload - Required parameters
+    # + return - Response or an error
+    remote isolated function checkContact(CheckContactRequestBody payload) returns CheckContactResponse|error {
+        string  path = string `/contacts`;
+        http:Request request = new;
+        json jsonBody = check payload.cloneWithType(json);
+        request.setPayload(jsonBody);
+        CheckContactResponse response = check self.clientEp->post(path, request, targetType=CheckContactResponse);
+        return response;
+    }
+    # Get-All-Groups
+    #
+    # + return - Response or an error
+    remote isolated function getAllGroups() returns GroupsResponse|error {
+        string  path = string `/groups`;
+        GroupsResponse response = check self.clientEp-> get(path, targetType = GroupsResponse);
+        return response;
+    }
+    # Create-Group
+    #
+    # + payload - Required parameters
+    # + return - Response or an error
+    remote isolated function createGroup(CreateGroupRequestBody payload) returns GroupsResponse|error {
+        string  path = string `/groups`;
+        http:Request request = new;
+        json jsonBody = check payload.cloneWithType(json);
+        request.setPayload(jsonBody);
+        GroupsResponse response = check self.clientEp->post(path, request, targetType=GroupsResponse);
+        return response;
+    }
+    # Get-Group-Info
+    #
+    # + GroupId - Group ID
+    # + return - Response or an error
+    remote isolated function getGroupInfo(string GroupId) returns GroupResponse|error {
+        string  path = string `/groups/${GroupId}`;
+        GroupResponse response = check self.clientEp-> get(path, targetType = GroupResponse);
+        return response;
+    }
+    # Update-Group-Info
+    #
+    # + GroupId - Group ID
+    # + payload - Required parameters
+    # + return - HTTP response or else an error
+    remote isolated function updateGroupInfo(string GroupId, UpdateGroupInfoRequestBody payload) returns http:Response|error {
+        string  path = string `/groups/${GroupId}`;
```

Comment:
@aneeshafedo isn't the path parameter variable name `GroupId` generate with camel cases? I mean something like `groupId`. @Chuhaa could you please check this. This is the case for the rest of the path parameters below in this class.

Reply:
This bug is already reported in [this](https://github.com/ballerina-platform/ballerina-openapi/issues/442) issue and fixed. Code generated from the latest version of the Ballerina Open API tool format `GroupId` to `groupId` as expected.

Reply:
Ok. Then we could regenerate the code for beta2 PR. I am merging all alpha5 PR that has a similar issue.